MeshLib/Elements/Element.h

```diff
@@ -67,7 +67,7 @@ class Element
      * modifiable by the user) instance of class Node or a NULL pointer
      * @sa Element::getNodeIndex()
      */
-    const Node* getNode(unsigned i) const;
+    Node* getNode(unsigned i) const;
```

Comment:
please keep the const and add a non-const overload:
`Node* getNode(unsigned i);`